File: 0128-longest-consecutive-sequence/0128-longest-consecutive-sequence.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Arrays.sort(nums);
        if (nums.length == 0) return 0;
        int c=1,m=1;
        for(int i=0;i<nums.length-1;i++)
        {
            if(nums[i+1]==nums[i])
            continue;
            else if(nums[i+1]==nums[i]+1)
            c++;
            else
            c=1;
            m=Math.max(m,c);
        }
        return m;
    }
}

File: 0128-longest-consecutive-sequence/0128-longest-consecutive-sequence.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
       
        l=0
        len=0
        num_s=set(nums)
        for num  in num_s:
            if num-1 not in num_s:
                curr=num
                len=1
                while curr+1 in num_s:
                    curr+=1
                    len+=1
            l=max(len,l)
        return l

        